package/index.js

```diff
@@ -84,6 +87,40 @@ function timeCalc(seconds) {
   return yDisplay + moDisplay + dDisplay + hDisplay + mDisplay + sDisplay;
 }
 
+// Area Formulas //
+
+//square
+function areaOfSquare(length) {
+  return length * length;
+}
+
+//Rectangle
+function areaOfRectangle(length, width) {
+  return length * width;
+}
+
+//Triangle
+function areaOfTriangle(base, height) {
+  return (1 / 2) * b * h;
+}
+
+//Circle
+function areaOfCircle(radius) {
+  return 3.14159 * radius * radius;
+}
+
+//Trapezoid
+function areaOfTrapezoid(base1, base2, verticalHeight) {
+  return (1 / 2) * (base1 + base2) * verticalHeight;
+}
+
+//Ellipse
+function areaOfEllipse(radiusmMajorAxis, radiusmMinorAxis) {
+  return 3.14159 * radiusmMajorAxis * radiusmMinorAxis;
+}
+
+// ======= End Of Area Formulas ========== //

```

Comment:
You're commenting here in wrong way. Use either `/*comment*/` or `//comment`.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix